PizzaOrder/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+</head>
+<body>
+    <p>Pizza Order</p>
+    <form action="bio.html">
+        <label for="name">Your name:</label>
+        <input type="text" name="name" minlength="3" maxlength="10" value="Michael Scott" required />
+
+        <br />
+
+        <label for="email">Your email:</label>
+        <input type="email" name="email" placeholder="example@example.com" pattern="[a-z]+@[a-z]+\.[a-z]{2,}" title="small letters without numbers. follow the example" required />
+        <br />
+
+        <label for="topping">Topping:</label>
+        <input type="radio" name="topping" value="supreme" checked />Supreme
+        <input type="radio" name="topping" value="vegetarian" checked />Vegetarian
+        <input type="radio" name="topping" value="hawaian" checked />Hawaian
```

Comment:
Aici as sugera sa fie schimbat astfel:

```suggestion
        <p>Topping:</p>
        <input type="radio" id="supreme" name="topping" value="supreme" checked /><label for="supreme">Supreme</label>
        <input type="radio" id="vegetarian" name="topping" value="vegetarian" checked /><label for="vegetarian">Vegetarian</label>
        <input type="radio" id="hawaian" name="topping" value="hawaian" checked /><label for="hawaian">Hawaian</label>
```
Astfel, fiecare input are legat un label.